project_issue_task_sync/models/project_task.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+
+
+class ProjectTask(models.Model):
+    _inherit = 'project.task'
+
+    issue_ids = fields.One2many('project.issue', 'task_id', 'Issues')
+
+    # CAN EXTEND THIS TO SYNC MORE FILES, if we want to create a model
+    # fields to sync we can do it here.
+
+
+    def get_changed_vals(self):
+        vals = {}
+        if self.issue_ids.stage_id != self.stage_id:
```

Comment:
protect against multiple issues: `[:1]`, as above, we should be able to assume those to have the same value anyways